pyqt-calculator-tutorial/examples/signals_slots_partial.py

```diff
@@ -0,0 +1,36 @@
+# Filename: signals_slots_partial.py
+
+"""Signals and slots example."""
+
+import sys
+import functools
+
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtWidgets import QWidget
+from PyQt5.QtWidgets import QPushButton
+from PyQt5.QtWidgets import QLabel
+from PyQt5.QtWidgets import QVBoxLayout
```

Comment:
I wonder if these imports should be in alphabetical order? I don't think our guide says anything about that as far as I can tell

Reply:
Same thing with `sys` and `functools` in that I would normally import `functools` first. But that's just me being nitpicky